connexion/utils.py

```diff
@@ -63,7 +63,10 @@ def deep_get(obj, keys):
     """
     if not keys:
         return obj
-    return deep_get(obj[keys[0]], keys[1:])
+    try:
```

Comment:
can you add a meaningful comment here? this code is not self-explanatory

Reply:
I updated the function's docstring. Does that add enough explanation?